request-processor/src/application/configs/mapping.yaml

```diff
@@ -184,3 +184,43 @@ mappings:
   description: "Reference must be unique"
   summary-singular: "1 reference must be unique"
   summary-plural: "{count} references must be unique"
+- field: conservation-area
+  issue-type: missing value
+  description: "Conservation area missing"
+  summary-singular: "1 conservation area missing"
+  summary-plural: "{count} Conservation areas missing"
```

Comment:
we should be consistent with capitalisation if it's user facing - i'm not sure if the c in conservation should be uppercase or lowercase but i know it shouldn't be a mixture